src/pod.ts

```diff
@@ -350,6 +351,24 @@ export class Pod {
     return this.cache.strings[options.value];
   }
 
+  /**
+   * Sets the environment using settings specified from one of the
+   * preconfigured environments in `amagaki.js`. This is useful for changing
+   * behavior depending on your build environment, such as outputting different
+   * content between dev, staging, and prod.
+   */
+  setEnvironment(name: string) {
+    if (!this.config.environments?.[name]) {
+      throw new Error(`Environment ${name} is not configured in amagaki.js.`);
+    }
+    this.env.name = name;
+    this.env.host = this.config.environments[name]?.host || this.env.host;
```

Comment:
Instead of using `this.config.environments[name]?`repeatedly, it could be simplified to `const env = this.config.environments[name]` and use `env.host`, etc below.